### PR TITLE
Allow selecting image for presto-admin product-tests

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: tests/docker_cluster.py [hunk 1]
```diff
@@ -190,8 +203,8 @@ def _create_and_start_containers(self, master_image, slave_image=None, cmd=None,
             self.master,
             hostname=self.internal_master,
             cmd=cmd,
-            links=zip(self.slaves, self.slaves),
             **kwargs)
+
```
Comment (kokosing) [inline tests/docker_cluster.py]: undo

File: tests/product/test_authentication.py [hunk 2]
```diff
@@ -31,7 +31,7 @@
 class TestAuthentication(BaseProductTestCase):
     def setUp(self):
         super(TestAuthentication, self).setUp()
-        self.setup_cluster(NoHadoopBareImageProvider, STANDALONE_PRESTO_CLUSTER)
+        self.setup_cluster(NoHadoopBareImageProvider(), STANDALONE_PRESTO_CLUSTER)
```
Comment (kokosing) [inline tests/product/test_authentication.py]: why?

File: tests/no_hadoop_bare_image_provider.py [hunk 3]
```diff
@@ -16,13 +16,16 @@
...
+import re
+
 from tests.bare_image_provider import TagBareImageProvider
 from tests.product.constants import BASE_IMAGES_TAG
 from tests.product.constants import BASE_IMAGE_NAME
 
 
 class NoHadoopBareImageProvider(TagBareImageProvider):
     def __init__(self):
+        decoration = 'nohadoop_' + re.sub(r"[^A-Za-z0-9]", "_", BASE_IMAGE_NAME)
```
Comment (kokosing) [inline tests/no_hadoop_bare_image_provider.py]: can you please explain a bit why it was needed?

File: tests/product/standalone/test_installation.py [hunk 4]
```diff
@@ -70,7 +69,7 @@ def test_install_non_root(self):
     @docker_only
     def test_install_on_wrong_os_offline_installer(self):
         image = 'teradatalabs/ubuntu-trusty-python2.6'
-        tag = BASE_IMAGES_TAG
```
Comment (kokosing) [inline tests/product/standalone/test_installation.py]: you can pull docker image here so you then, you could remove `make docker-images`

File: tests/product/constants.py [hunk 5]
```diff
@@ -15,29 +15,30 @@
...
+BASE_IMAGE_TAG = os.environ.get('BASE_IMAGE_TAG')
+
+if BASE_IMAGE_NAME is None:
+    BASE_IMAGE_NAME = 'teradatalabs/centos6-ssh-oj8'
+
+if BASE_IMAGE_TAG is None:
+    try:
+        with open(os.path.join(main_dir, BASE_IMAGES_TAG_CONFIG)) as tag_config:
+            tag_json = json.load(tag_config)
+        BASE_IMAGE_TAG = 'latest'
```
Comment (kokosing) [inline tests/product/constants.py]: use `tag_json`

File: bin/ci-product.sh [hunk 6]
```diff
@@ -12,5 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-make test-images
```
Comment (kokosing) [inline bin/ci-product.sh]: maybe lest remove this from Makefile

Comment (losipiuk) [reply bin/ci-product.sh]: Apparently this is needed. We looked at that with @sopel39 and decided removing that would require understanding large part of this crap. So let's leave that.

File: tests/product/standalone/test_installation.py (hunk 4 above)
Comment (losipiuk) [reply tests/product/standalone/test_installation.py]: not 100% sure leaving for now until it works

File: tests/product/test_authentication.py (hunk 2 above)
Comment (losipiuk) [reply tests/product/test_authentication.py]: We pass instance now

**Pull request merged**